src/docfx/build/xref/XrefMap.cs

```diff
@@ -48,7 +48,7 @@ public static async Task<XrefMap> Create(Context context, Docset docset)
                     map[sepc.Uid] = sepc;
                 }
             }
-            return new XrefMap(map, docset.Config.BuildInternalXrefMap ? await InternalXrefMap.Create(context, docset.BuildScope) : null);
+            return new XrefMap(map, docset.Config.BuildInternalXrefMap ? InternalXrefMap.Create(context, docset.BuildScope) : null);
```

Comment:
Feels like a simple function `CreateInternalXrefMap() -> IReadOnlyDictionary<string, XrefSpec>` is enough